web/themes/custom/vote_gov/templates/block/block-content--basic.html.twig

```diff
@@ -0,0 +1,27 @@
+
+
+{% block content %}
+{% set classes = [ 
+    content.field_classes | field_value | render
+] %}
+
+<div{{ attributes.addClass(classes) }}>
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-4">
+      </div>
+      <div class="tablet:grid-col-8 right-shift">
+            <section class="page-section registered" id="already-registered">
```

Comment:
Remove the id attribute. If there are mulitple basic blocks on a page this will throw an accessibility error.